R/helper_lm_robust_fit.R

```diff
@@ -241,9 +241,9 @@ lm_robust_fit <- function(y,
 
       }
     } else if (weighted) {
-      return_list[["fitted.values"]] <- fit_vals[["fitted.values.unweighted"]]
+      return_list[["fitted.values"]] <- as.matrix(fit_vals[["fitted.values.unweighted"]])
```

Comment:
This reads as "if it's weighted, return the unweighted fits" ?

Reply:
Yes, we always return unweighted fits.